No-Ones-Jupyter-Notebook.ipynb

```diff
@@ -1,15 +1,27 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "This notebook contains Julia code used to generate results which will be published in the `Numbers Without Ones (Revisited)` article in the `Recreational Mathematics Magazine`.\n",
```

Comment:
How about this? "This notebook contains Julia code used to generate examples and tables in the article `Numbers Without Ones (Revisited)`, *Recreational Mathematics Magazine* (forthcoming 2023). \n",

Reply:
OK I think I fixed this now.